Google SSO updates (#2622)
mattermost/docs: source/deployment/sso-google.md

**Note: Google SSO is not yet supported in the mobile app. If you want to use Google SSO on mobile, you'll have to [build your own app](https://developers.mattermost.com/contribute/mobile/build-your-own/).**

1 - Go to [Google API Manager](https://console.developers.google.com), click **Credentials** in the left hand sidebar, click **Create a project** and enter "Mattermost - your-company-name" as the **Project Name**, then click **Create**.

![google_1_credentials](../../source/images/google_1_credentials.png)

2 - In the **Dashboard** click the option to **ENABLE APIS AND SERVICES**.

![google_enable_api](../../source/images/google_enable_api.png)

Once in the API Library, scroll down to [Google+ API](https://console.developers.google.com/apis/api/plus/overview) and click **Enable** in the header. This might take a few minutes to propagate through Google's systems.

> 3 - Leave the Google+ API menu and return to the **APIs & Services menu** and in the left hand sidebar select **Credentials**, click the **OAuth consent screen** header and enter "Mattermost" as the **Product name shown to users**, then click **Save**.

Reviewer: ```suggestion
3 - Leave the Google+ API menu to return to the **APIs & Services menu**. In the left hand sidebar select **Credentials**, click the **OAuth consent screen** header and enter "Mattermost" as the **Product name shown to users**, then click **Save**.
```

![google_2_oauth_consent_screen](../../source/images/google_2_oauth_consent_screen.png)

> Scroll further down and you will need to add the domain used by your Mattermost instance to a list that is authorised to access the API. For example, if it is hosted on `mattermost.yourdomain.com` you would add `yourdomain.com`.

Reviewer: ```suggestion
Scroll further down to add your Mattermost instance to a list of authorized domains that can access the API. For example, if it is hosted on `mattermost.yourdomain.com` add `yourdomain.com`.
```
